server/controllers/groups.js

```diff
@@ -127,11 +127,11 @@ export const getTransactions = (req, res, next) => {
     GroupId: req.group.id
   };
 
-  if (req.query.donation) {
+  if (req.query.donation || req.query.type === 'donations') {
```

Comment:
This is needed for https://github.com/OpenCollective/opencollective-website/pull/617